streamlit_chat/frontend/src/stChat.tsx

```diff
@@ -71,7 +72,7 @@ class Chat extends StreamlitComponentBase {
     return (
       <Chat isUser={isUser}>
         <Avatar src={avatarUrl} alt="profile" draggable="false"/>
-        <Message>{message}</Message>
+        <Message>{ReactHtmlParser(message)}</Message>
```

Comment:
Can you add a parameter which will enable/disable this, like the allow_unsafe_html parameter

Reply:
@yashppawar 
As your suggestion, I've added `allow_html` parameter.